_posts/2016-05-16-rust-at-one-year.md

```diff
@@ -0,0 +1,269 @@
+  when recompiling code after editing it. A vital step here was the move to a
+  [custom intermediate representation][MIR], which has many other benefits as
+  well. Another focus has been errors, including
+  [detailed explanations of most errors][error index], and ongoing work to
+  [improve the clarity and focus of errors][error format]. Expect to hear more
+  on both fronts soon.
+
+[across-the-board improvements]: http://blog.rust-lang.org/2015/06/25/Rust-1.1.html
+[parallel codegen]: http://blog.rust-lang.org/2015/08/06/Rust-1.2.html
+[incremental compilation]: https://github.com/rust-lang/rfcs/pull/1298
+[MIR]: http://blog.rust-lang.org/2016/04/19/MIR.html
+[error index]: https://doc.rust-lang.org/error-index.html
+[error format]: https://internals.rust-lang.org/t/new-error-format/3438
+
+- **Core language**. We've kept one list purposefully short this year: growth in
+  the core language. While we have some important features in the pipeline (like
+  [improved error handling], [more flexible borrowing rules] and
+  [specialization]), [Rust users] by and large are happy with the core language
+  and prefer the community to focus on the ecosystem and tooling.
+
+[Rust users]: https://internals.rust-lang.org/t/production-user-research-summary/2530
+[improved error handling]: https://github.com/rust-lang/rfcs/pull/243
+[more flexible borrowing rules]: http://smallcultfollowing.com/babysteps/blog/2016/04/27/non-lexical-lifetimes-introduction/
+[specialization]: https://github.com/rust-lang/rfcs/pull/1210
+
+There's a lot more to say about what's happened and what's coming up in the Rust
+world&mdash;over the coming months, we'll be using this blog to say it.
+
+### Rust in community
+
+It turns out that people like to get together and talk Rust. We had a sold out
+[RustCamp] last August, and several upcoming events in 2016:
+
+- September 9-10, 2016: the first [RustConf] in Portland, OR, USA;
+- September 17, 2016: [RustFest], the European community conference, in Berlin, Germany;
+- October 27-18, 2016: [Rust Belt Rust], a Rust conference in Pittsburgh, PA, USA;
+- 71 Rust-related [meetup] groups worldwide.
+
+[RustCamp]: http://rustcamp.com/
+[RustConf]: http://rustconf.com/
+[RustFest]: http://www.rustfest.eu/blog/happy-birthday-announcing-rustfest
+[Rust Belt Rust]: http://rust-belt-rust.com/
+[meetup]: http://rust.meetup.com/
+
+And that's no surprise. From a personal perspective, the best part about working
+with Rust is its [community]. It's hard to explain quite what it's like to be
+part of this group, but two things stand out. First, its sheer *energy*: so much
+happens in any given week that [This Week in Rust] is a vital resource for
+anyone hoping to keep up. Second, its *welcoming spirit*. Rust's core message is
+one of empowerment&mdash;you can write systems code even if you would never have
+dared write C++&mdash;and that's reflected in the community. We're all here to
```

Comment:
"never have dared write C++" as similar confrontational feel.  Could we say "you can write safe systems code more easily than was previously possible"?